### ISSUE TITLE
Attempt to invoke virtual method 'java.lang.String com.facebook.AccessToken.getApplicationId()' on a null object reference

### ISSUE THREAD

**Issue by Symyon** [facebook/react-native-fbsdk#456]
## Environment

React Native Environment Info:
    System:
      OS: macOS 10.14.2
      CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
      Memory: 309.53 MB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 8.14.0 - /usr/local/opt/node@8/bin/node
      Yarn: 1.12.3 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/opt/node@8/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 25, 26, 27, 28
        Build Tools: 28.0.3
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.5.0 => 16.5.0
      react-native: https://github.com/expo/react-native/archive/sdk-31.0.1.tar.gz => 0.57.1
    npmGlobalPackages:
      react-native-cli: 2.0.1

## Description

On Android (iOS works), when calling a GraphRequest "/page-id/feed" and the user is not logged in with Facebook I get a null reference. This is because the setConfig() method from FBGraphRequestModule.java, when setting the accessToken is is trying to use (read) appId and userId from AccessToken.getCurrentAccessToken() which is null (we have no Facebook user logged in). Check line 157/158
<img width="707" alt="screen shot 2019-01-09 at 3 51 44 pm" src="https://user-images.githubusercontent.com/1674549/50927707-8d183f00-1426-11e9-85d6-6f7a1ad3a096.png">

## Reproducible Demo

Not sure how I can share a Demo about it.

**Comment by hyebhinkim** [comment]
i need your help 
message me plz
hyebin0319@gmail.com

**Comment by superandrew213** [comment]
Can confirm this. @Symyon did you find a workaround?

**Comment by Symyon** [comment]
@superandrew213 @hyebhinkim  yes, but it's dirty. I just set a default string for appId and user ID if access token is null. Ex:
`AccessToken at = AccessToken.getCurrentAccessToken();
String appId = at == null ? "deafultAppId" : at.getApplicationId();
String userId = at == null ? "defaultUserId" : at.getUserId();
graphRequest.setAccessToken(new AccessToken(
appId,
userId,
.......`
